src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -8115,22 +8148,75 @@ export class StudyViewPageStore
         } else return '';
     }
 
+    readonly survivalEntryMonths = remoteData<
+        { [uniquePatientKey: string]: number } | undefined
+    >({
+        invoke: async () => {
+            const studyIds = this.studyIds;
+            // Please note:
+            // The left truncation adjustment is only available for one study: heme_onc_nsclc_genie_bpc at this time
+            // clinical attributeId still need to be decided in the future
+            if (
+                studyIds.length === 1 &&
+                studyIds[0] === 'heme_onc_nsclc_genie_bpc' &&
+                this.enableLeftTruncationForSuvivalData
```

Comment:
typo in that name "Suvival" also, if this is a boolean, lets name it isLeftTruncationForSurvivalDataEnabled.    "enable" is very making me think this is an action and not a piece of state.

Reply:
Comments addressed in https://github.com/cBioPortal/cbioportal-frontend/pull/4324/commits/1359463a4b53e54a0530bdb509f659ebb929368b